languages.yml

```diff
@@ -0,0 +1,10 @@
+# http://userguide.icu-project.org/locale#TOC-Language-code
+# Feel free to add your locale and translation, the more languages we have the more awesome multilingual hugs can be!
+languages:
+  de: Umarmen.
+  se: Kram!
```

Comment:
`da: Knus!` maybe? 🤗

Reply:
Can't believe I forgot Danish! Thanks for the reminder, added! *knus*